geoCore/config/config.yml

```diff
@@ -1 +1,16 @@
-layerSchichtdaten: Schichtdaten
\ No newline at end of file
+layerSchichtdaten: Schichtdaten
+
+# columns in borehole data
+xCoord: xcoord
+yCoord: ycoord
+zCoord: zcoorddb
+
+# columns in layer data
+layerNo: schichtnr
+group: gruppierung
+depthFrom: tiefe von
+depthTo: tiefe bis
+petrography: petrographie
+color: farbe
+facies: facies
+comment: beschreibung
```

Comment:
IMO you should always add one blank line to all files...

Reply:
You're absolutely right. This was an oversight and has been corrected.